extensions/amp-render/1.0/amp-render.js

```diff
@@ -132,7 +147,12 @@ export const getJsonFn = (element) => {
   if (isAmpScriptSrc(src)) {
     return (src) => getAmpScriptJson(element.getAmpDoc(), src);
   }
-  return () => batchFetchJsonFor(element.getAmpDoc(), element);
+  return (_src, shouldRefresh = false) =>
```

Comment:
Is the intention with naming with the `_` prefix that it will be unused?
```suggestion
  return (unusedSrc, shouldRefresh = false) =>
```

Reply:
Yes, it's what prettier recommended. Anyways, updated it to `unusedSrc`.